addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2659,14 +2659,36 @@ export class OdooEditor extends EventTarget {
         };
 
         const side1 = isRow ? 'left' : 'top';
+        const side2 = isRow ? 'top' : 'left';
+        const scrollContainerRect = this.options.getScrollContainerRect();
+
         ui.style[side1] = (isRow ? elementRect : tableRect)[props.xy[side1]] - togglerRect[props.size[side1]] + 'px';
-        ui.style[props.size[side1]] = !isRow && togglerRect[props.size[side1]] + 'px';
+        if (tableRect.top > scrollContainerRect.top) {
+            ui.style[props.size[side1]] = !isRow && togglerRect[props.size[side1]] + 'px';
+            if (tableRect[side1] - scrollContainerRect.top < 15 && !isRow) {
```

Comment:
Why 15? where are we getting this value from?

Reply:
15 was the height of the table menu toggle. But I encountered some issues while testing which led me to removal of this code.